Dockerfile

```diff
@@ -68,26 +68,26 @@ RUN wget https://github.com/koalaman/shellcheck/releases/download/v$SHELLCHECK_V
     tar xJvf - shellcheck-v$SHELLCHECK_VERSION/shellcheck          && \
     mv shellcheck-v$SHELLCHECK_VERSION/shellcheck /bin             && \
     rmdir shellcheck-v$SHELLCHECK_VERSION
-RUN shellcheck -V
-
-# Set locale
-RUN locale-gen en_US.UTF-8
 
 # Print tool versions
-RUN bash --version | head -n 1
-RUN zsh --version
-RUN ksh --version || true
-RUN dpkg -s dash | grep ^Version | awk '{print $2}'
-RUN git --version
-RUN curl --version
-RUN wget --version
+RUN bash --version | head -n 1 \
+    && zsh --version \
+    && ksh --version || true \
+    && dpkg -s dash | grep ^Version | awk '{print $2}' \
+    && git --version \
+    && curl --version \
+    && wget --version \
+    && shellcheck -V
 
 # Add user "nvm" as non-root user
 RUN useradd -ms /bin/bash nvm
 
+# Set NVM root dir
+ARG NVM_DIR=/home/nvm
```

Comment:
why is `.nvm` removed?

Reply:
This was my mistake. I forgot to add it back in. 

The advantage of using AGR is that you just have to change one line and can even change it via CLI argument during the build time.

Reply:
Yeah I understand the advantage, looks fine to me.